trento/migration/container-install.md

```diff
@@ -54,7 +54,7 @@ Follow the steps in [4.2 systemd deployment](https://documentation.suse.com/sles
 
 1. Create the secret environment variables:
 
-   > Note: Consider using an environment variable file, learn more about from [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env). Adjust upcoming commands for env file usage.
+   > Note: Consider using an environment variable file, learn more about from [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env). Adjust upcoming commands for env file usage. Regardless, make sure you keep a copy of the generated keys in a safe location, in case you need to reutilize them in the future. 
```

Comment:
What do you think about shortening it a bit up ?
Examples:
- Keep a copy of the generated keys in a safe location, in case you need to reutilize them in the future.
- Additionally, keep a copy of the generated keys in a safe location, in case you need to reutilize them in the future.
- Regardless keep a copy of the generated keys in a safe location, in case you need to reutilize them in the future.

Reply:
Any of your suggestions LGTM. Pick up the one you like the most :-)